### PR TITLE
chore: update web3 to 1.2.4

### PR DESCRIPTION
**Pull request by e00dan**

### CODE REVIEW

File: packages/origin/src/test/CertificateLogic.test.ts [hunk 1]
```diff
@@ -546,7 +539,7 @@ describe('CertificateLogic-Facade', () => {
 
         const certificate = await new Certificate.Entity('6', conf).sync();
 
-        blockCreationTime = (await web3.eth.getBlock('latest')).timestamp;
+        blockCreationTime = parseInt((await web3.eth.getBlock('latest')).timestamp.toString(), 10);
```
Comment (kosecki123) [inline packages/origin/src/test/CertificateLogic.test.ts:542]: What is the reason for parse(string), https://web3js.readthedocs.io/en/v1.2.1/web3-eth.html#getblock timestamp should be number already.

File: packages/origin/src/blockchain-facade/Certificate.ts [hunk 2]
```diff
@@ -131,14 +133,21 @@ export const getAllCertificateEvents = async (
...
+
     public energy: number;
+
     public status: Status;
+
     public creationTime: number;
+
     public parentId: number;
+
     public children: string[];
```
Comment (kosecki123) [inline packages/origin/src/blockchain-facade/Certificate.ts:151]: Are these whitespace add manually or we somehow have different prettier configs?

**Review by kosecki123 — APPROVED** [review]

File: packages/origin/src/blockchain-facade/Certificate.ts (hunk 2 above)
Comment (josipbagaric) [reply packages/origin/src/blockchain-facade/Certificate.ts:151]: Yeah this is added automatically, not a fan of this either. We should look into removing it IMO.

File: packages/user-registry/package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     "test": "test"
   },
   "scripts": {
-    "build": "yarn build:ts",
+    "build": "yarn build:static && yarn build:ts",
```
Comment (josipbagaric) [inline packages/user-registry/package.json:25]: Nice one. I think we should change `yarn build` in every repo to behave like this.

**Review by josipbagaric — APPROVED** [review]

File: packages/origin/src/test/CertificateLogic.test.ts (hunk 1 above)
Comment (e00dan) [reply packages/origin/src/test/CertificateLogic.test.ts:542]: Unfortunately docs aren't up to date.

typess

**Pull request merged**